web/blog/2023-10-17-from-candidate-to-colleague.md

```diff
@@ -0,0 +1,604 @@
+---
+title: '👥 From Candidate to Colleague: Acing a competitive Junior-level recruitment challenge 📋'
+authors: [lucas]
+image: img/from-candidate-to-colleague.gif
+tags: [career, web-development, tutorial, learning]
+---
+
+## Hello there!
+
+## TL;DR
+
+![Project Overview](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/9g5owahcqzhpo331wol3.png)
+
+In this article, we want to emphasize the importance of coding challenges by giving you the skills you’ll need to pass a common junior-level interview challenge: building a full-stack CRUD app (which we will use the [Wasp framework](https://wasp-lang.dev/) and it’s stack — React, Node and Prisma to do so). The application focuses on making a web app for trading and rating Pokémon based on their Experience Points (XP), using the [PokéAPI](https://pokeapi.co/) to get the data that we need.
+
+We will also discuss the important factors that recruiters look for when hiring new developers, such as code quality, readability, stack and architectural choices and a more modern approach to AI use in code generation. It's a helpful read if you want to improve your coding skills and succeed in the hiring process!
+
+You can find the deployed version of the app [here](https://client-production-55c6.up.railway.app/)!
+
+### Before we Begin
+
+![funny gif](https://media1.giphy.com/media/QF0qzNUO8FfX2/giphy.gif?cid=7941fdc6xrdr0zrg9zteorw0fxnkjz5q6ony465zf9uekqnx&ep=v1_gifs_search&rid=giphy.gif&ct=g)
+
+In this article, I'll be using the [Wasp Framework](https://github.com/wasp-lang/wasp). It provides a great way to start a project with its AI app generator and a really awesome developer experience for when you want to create a full-stack web application with all batteries included, front-end, back-end, and database.
+
+I'm currently a member of the Wasp team and since it’s also an open-source project, you can contribute too!
+
+The easiest way to show your support is just to star our repo! 🐝 But it would be greatly appreciated if you could take a look at our repository (for contributions or to simply test the product) too!
+
+{% cta https://www.github.com/wasp-lang/wasp %} ⭐️ Give Wasp Framework a Star! ⭐️ {% endcta %}
```

Comment:
dev.to specific. this needs to be turned into a link.

Reply:
Nice catch! Fixed 👍